udev/69-hid.rules

```diff
@@ -0,0 +1,33 @@
+# This is a sample udev file for HIDAPI devices which lets unprivileged
+# users who are physically present at the system (not remote users) access
+# HID devices.
+
+# If you are using the libusb implementation of hidapi (libusb/hid.c), then
+# use something like the following line, substituting the VID and PID with
+# those of your device.
+
+# HIDAPI/libusb
+SUBSYSTEM=="usb", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="003f", TAG+="uaccess"
+
+# If you are using the hidraw implementation (linux/hid.c), then do something
+# like the following, substituting the VID and PID with your device.
+
+# HIDAPI/hidraw
+KERNEL=="hidraw*", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="003f", TAG+="uaccess"
+
+# Once done, optionally rename this file for your application, and drop it into
+# /etc/udev/rules.d. Note that these rules must have priorty before 70-uaccess.rules
```

Comment:
@Be-ing I hope you do not mind the tag.  Can you help explain the following:

> that these rules must have priorty before `70-uaccess.rules`

Reply:
I don't recall the reason for this. I'm not sure we ever dug deep to figure out exactly what the issue was, but IIRC we stumbled on this shipping the udev rule file for Mixxx.

Reply:
If you figure this out, it would be great if you open a pull request to clarify the documentation in this example udev rule file.

Reply:
I've been having problems using the `hidraw` in general so I came here more as a "what am I doing wrong" step.  I'll surely improve if I get further. 🍺

Reply:
Is this for LMMS?

Reply:
It's actually for another project (QZ Tray), which uses hidapi through hid4java.

Reply:
> Can you help explain the following:
>>that these rules must have priorty before `70-uaccess.rules`

I'm pretty sure it is bacause of the use `TAG+="uaccess"` - this tag is handled by the `70-uaccess.rules` file, and all the rule files are processed in strict order, and if the tag is added _after_ `70-uaccess.rules` is processed - the device will be processed as if this tag is not present at all.

Reply:
@Youw thanks for the tip... I stumbled upon this bug report which seems to state something similar. 
 https://github.com/systemd/systemd/issues/4288#issuecomment-348166161

Reply:
Aha, so I haven't actually looked up the documentation for `70-uaccess.rules`, I just assumed its purpose based on the documentation as suggested by this PR.
Thanks for pointing out to that comment. It appears my assumption is absolutely correct, but regarding a different file: `73-seat-late.rules`.

I'll update a documentation once again.

Reply:
On that note: it looks like [`70-uaccess.rules`](https://cgit.freedesktop.org/systemd/systemd/tree/src/login/70-uaccess.rules) just automatically adds the `uaccess` tag to a few known devices/device types.